frontend/packages/erd-core/src/schemas/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './cliVersion'
+export * from './queryParam'
+export * from './showMode'
```

Comment:
I only wanted the cliVersion here, but I'm also exporting the others for consistency